pkg/plugin/integration_test.go

```diff
@@ -272,8 +274,49 @@ func setupNewCluster(ctx context.Context, log *logrus.Entry, cs *api.OpenShiftMa
 				},
 			},
 		}
+
 		return true, node, nil
 	})
+	cli.PrependReactor("get", "cronjobs", func(action k8stesting.Action) (bool, runtime.Object, error) {
+		get := action.(k8stesting.GetAction)
+		cronjob := &batchv1b1.CronJob{
```

Comment:
In my experience, Kube cronjobs are notoriously unreliable on failures and most applications that they seemed right for we ended up using one-shot pods. Is this something we've considered? What's our error-handling story if this fails?

Reply:
> In my experience, Kube cronjobs are notoriously unreliable on failures and most applications that they seemed right for we ended up using one-shot pods. Is this something we've considered? What's our error-handling story if this fails?

This PR is not introducing cronjobs, just adding the mocking for it.
But I don't believe when I did add it anyone else had a bad experience with them, else i would have done something different. They appear to be running just fine though... We were going to have some kind of monitoring to detect if the backups failed (or failed to happen - not sure if this is in place)..

Reply:
This is fine, just wanted to mention it :) we were seeing something like 20% error rates on cronjobs failing (and they weren't retrying) at my last gig.

Reply:
Cronjobs are very sensitive to their config. took us a while to refactor our retry config in CI from purge jobs too. Agreed :)